composites/AlgoliaSearch/AlgoliaSearcher.js

```diff
@@ -51,29 +73,38 @@ class AlgoliaSearcher extends React.Component {
 	/**
 	 * Handles the form submit event. Stores the search string and performs a search.
 	 *
-	 * @param {object} clickEvent The React SyntheticEvent.
+	 * @param {string} searchString The entered search string.
 	 *
 	 * @returns {void}
 	 */
-	searchButtonClicked( clickEvent ) {
-		let searchString = clickEvent.target.getElementsByTagName( "input" )[ 0 ].value;
-
+	onSearchButtonClick( searchString ) {
```

Comment:
pending decision on the button, see #285, there's no button right now. The function name should reflect what the function does.